### DISCUSSION TITLE
✅ Image from an external URL

### DISCUSSION THREAD

**Discussion by LeiWart** [elementor/elementor#24236] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

To avoid endless duplication and loading of images from one service to another, you can implement the function of image loading by external URL passed to the Image module.

### Describe the solution you'd like

The Image Elementor module accepts an external URL and loads the specified image onto the page.

### Additional context

I wanted to create a custom plugin that will take parameters from the URL on which a person goes, using these parameters to load data from another service, including the URL of the picture, and pass the URL to the Image field on the site. Thus, a person who follows a specialized link will receive information about a particular record from another source and will also be able to see the image attached to this record.

**Comment by nicholaszein** [comment]
Hi. This feature already exists. When selecting the image, just find the option **"Insert from URL"**.

<img width="770" height="278" alt="image" src="https://github.com/user-attachments/assets/3499e656-9190-48c2-acec-b8e41228e8a6" />


Kind regards
